hardware/CommunicationsPi/comm_pi.py

```diff
@@ -1,10 +1,12 @@
 from http.server import BaseHTTPRequestHandler
 from hardware.CommunicationsPi.radio_transceiver import Transceiver
 
-transceiver = Transceiver()
-
 
 class CommPi(BaseHTTPRequestHandler):
+    def __init__(self, *args, **kwargs):
+        self.transceiver = Transceiver()
+        super().__init__(*args, **kwargs)
```

Comment:
I think super calls must be before any other initialization. I learned this from Java and javascript, not sure about python

Reply:
I thought the same, but for some reason when I had the super call before, the transceiver was never initialized.